src/ui/server/routes/insurance/index.ts

```diff
@@ -47,4 +49,10 @@ insuranceRouter.use(INSURANCE_ROOT, insuranceCheckYourAnswersRouter);
 
 insuranceRouter.get(`${INSURANCE_ROOT}/:referenceNumber${INSURANCE_ROUTES.APPLICATION_SUBMITTED}`, applicationSubmittedGet);
 
+insuranceRouter.get(INSURANCE_ROUTES.FEEDBACK, feedbackGet);
+insuranceRouter.post(INSURANCE_ROUTES.FEEDBACK, feedbackPost);
+
+insuranceRouter.get(INSURANCE_ROUTES.FEEDBACK_CONFIRMATION, feedbackConfirmationGet);
+insuranceRouter.post(INSURANCE_ROUTES.FEEDBACK_CONFIRMATION, feedbackConfirmationPost);
```

Comment:
Do we need a POST for the sucess/confirmation page? If there is a button link perhaps it could just be a button link?